src/filesystem-upload-options.js

```diff
@@ -29,6 +45,19 @@ export default class FileSystemUploadOptions extends DirectBinaryUploadOptions {
         const newOptions = new FileSystemUploadOptions();
         newOptions.options = { ...uploadOptions.options };
         newOptions.controller = uploadOptions.controller;
+
+        if (uploadOptions.replaceValue !== undefined) {
```

Comment:
If `replaceValue` is null, we'll get into the if. Is this wanted behavior?

Reply:
Good catch - I've updated the logic.